module.js

```diff
@@ -9,6 +9,8 @@ export default function(moduleOptions) {
     ...this.options.forecastViz
   }
 
+  if (!options.disclaimer) options.disclaimer = ''
+  const { disclaimer } = options
```

Comment:
I think we probably don't need this line because we're not using the disclaimer value anywhere in this function, right?
```suggestion
```